tensorrt_llm/_torch/custom_ops/trtllm_gen_custom_ops.py

```
@@ -97,31 +101,22 @@ def __eq__(self, other):
         if instance_key not in FP4BlockScaleMoERunner.runner_dict:
             FP4BlockScaleMoERunner.runner_dict[
-                instance_key] = torch.classes.trtllm.FP4BlockScaleMoERunner(
-                    tile_tokens_dim)
+                instance_key] = torch.classes.trtllm.FP4BlockScaleMoERunner()
         return FP4BlockScaleMoERunner.runner_dict[instance_key]
 
     def forward(
         self,
         inputs: List[torch.Tensor],
-        tactic: int = -1,
+        tactic: List[int] = [-1, -1],
```

Ruff (0.14.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)